src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -382,7 +382,7 @@ def forward(
             layer_head_mask (`torch.FloatTensor`): mask for attention heads in a given layer of size
                 `(encoder_attention_heads,)`.
             cross_attn_layer_head_mask (`torch.FloatTensor`): mask for cross-attention heads in a given layer of
```

Comment:
```suggestion
            layer_head_mask (`torch.FloatTensor` of size `(encoder_attention_heads,)`): 
                mask for attention heads in a given layer
            cross_attn_layer_head_mask (`torch.FloatTensor`):
                mask for cross-attention heads in a given layer of
```
Well we changed the way the doc is so just putting this because we could update the MBart doc

Reply:
Thanks for the details! I'll save this for a follow-up PR